tests/test_fundamental_params.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+from sedkit import sed
+import astropy.units as u
+import numpy as np
+
+
+@pytest.fixture
+def sub_spec():
+    # Initialize the substellar SED
+    sub_spec = sed.SED(name="2MASS J04151954-0935066", substellar=True)
+    # Load in a spectrum to test
+    sub_spec.add_spectrum_file(
+        "tests/data/2MASS_J04151954-0935066_apparent_SED.txt",
+        wave_units=u.micron,
+        flux_units=u.erg / u.s / u.cm**2 / u.AA,
+    )
+    return sub_spec
+
+
+@pytest.fixture
+def spec():
+    # Initialize the non_substellar SED
+    spec = sed.SED(name="2MASS J04151954-0935066", substellar=False)
+    # Load in a spectrum to test
+    spec.add_spectrum_file(
+        "tests/data/2MASS_J04151954-0935066_apparent_SED.txt",
+        wave_units=u.micron,
+        flux_units=u.erg / u.s / u.cm**2 / u.AA,
+    )
+    return spec
+
+
+@pytest.mark.parametrize("seds", ["sub_spec", "spec"])
+def test_just_spectrum(seds, request):
+    sed = request.getfixturevalue(seds)
+    sed.spectral_type = None    # forcing the SED to take no info from SIMBAD on SPT
+    sed.distance = None     # forcing the SED to take no info from SIMBAD on distance
+
+    sed.results
+    assert np.isclose(sed.fbol[0], 1.9184645e-12 * u.erg / u.s / u.cm**2)
+    assert np.isclose(sed.fbol[1], 5.26164965e-15 * u.erg / u.s / u.cm**2)
+    assert sed.mbol == (17.811, 0.003)
+    assert sed.Lbol is None
+    assert sed.Lbol_sun is None
+    assert sed.radius is None
+    assert sed.Teff is None
+    assert sed.logg is None
+    assert sed.mass is None
+
+
+@pytest.mark.parametrize(
+    "seds,radius_expected,teff_expected",
+    [
+        (
+            "sub_spec",
+            (0.973 * u.Rjup, 0.0 * u.Rjup, 0.0 * u.Rjup),
```

Comment:
Is this what you really want, uncertainties of 0?

Reply:
I think it's happening because of the `round()` statements on line 289 of relations.py.  See my other comment.